test/purging.sh.in

```diff
@@ -128,26 +128,51 @@ test_result $?
 
 echo
 echo
-echo " == preceding RMWTRASH=empty with the rmw command should completely"
-echo " == purge the waste folders, no matter when they were deleted"
+echo " == rmw -e command should completely purge the waste folders,"
+echo " == no matter when they were deleted"
 
 $RMW_TEST_CMD_STRING --verbose $HOME/somefiles
 test_result $?
 
 echo " == Should not work if '-f' isn't used"
-output=`RMWTRASH=empty $RMW_ALT_TEST_CMD_STRING --purge`
+output=`echo "y" | $RMW_ALT_TEST_CMD_STRING --purge -e`
 test "$output" = "purge has been skipped: use -f or --force"
 test_result $?
 
-echo " == Should not work if '--purge' isn't used"
-output=`RMWTRASH=empty $RMW_TEST_CMD_STRING -f`
-test "$output" = "'-f/--force' does nothing without '-g/--purge'"
+# echo " == Should not work if '--purge' isn't used"
+# output=`echo "y" | $RMW_ALT_TEST_CMD_STRING -ffe`
+# test "$output" = "'-f/--force' does nothing without '-g/--purge'"
+# test_result $?
+
+echo " == Should not work 'Y' or 'y' is not supplied."
+output=$(echo `echo "yfw" | $RMW_TEST_CMD_STRING --purge -e`)
+test "$output" = "All the files in trash will be unrecoverable after empty. Continue? (y/n): Action cancelled."
+test_result $?
+
+output=$(echo `echo "Yfw" | $RMW_TEST_CMD_STRING --purge -e`)
+test "$output" = "All the files in trash will be unrecoverable after empty. Continue? (y/n): Action cancelled."
+test_result $?
+
+output=$(echo `echo "n" | $RMW_TEST_CMD_STRING --purge -e`)
+test "$output" = "All the files in trash will be unrecoverable after empty. Continue? (y/n): Action cancelled."
+test_result $?
+
+output=$(echo `echo "Nqer" | $RMW_TEST_CMD_STRING --purge -e`)
+test "$output" = "All the files in trash will be unrecoverable after empty. Continue? (y/n): Action cancelled."
+test_result $?
+
+echo " == Empty works with force empty (-ffe)"
+echo "y" | $RMW_TEST_CMD_STRING -ffe
 test_result $?
 
-echo " == Should not work if '--purge' and '-f' isn't used"
-RMWTRASH=empty $RMW_TEST_CMD_STRING
+echo " == Should work with 'Y' or 'y'"
+echo "y" | $RMW_TEST_CMD_STRING -e
 test_result $?
 
+echo "Y" | $RMW_TEST_CMD_STRING -e
+test_result $?
```

Comment:
I think in a test env it may be a good idea to have a test for `./rmw -e` and `user_verify()`, because both functions are linked. What I mean is:

1. A coder one day may decide to modify `user_verify()` and its associated test, but they may not realise what the consequences of modifying this function will be.
2. Tests show up what modifying `user_verify()` causes to fail. The coder can then easily pinpoint what needs fixing.

Reply:
ok